### PR TITLE
feat(slack): Implement assign to teams

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/integrations/slack/utils.py [hunk 1]
```diff
@@ -5,35 +5,43 @@
...
 NEW_ISSUE_COLOR = '#E03E2F'
 ACTIONED_ISSUE_COLOR = '#EDEEEF'
 
 
-def get_assignees(group):
+def format_actor_option(actor):
+    if isinstance(actor, User):
+        return {'text': actor.get_display_name(), 'value': u'user:{}'.format(actor.id)}
+    if isinstance(actor, Team):
+        return {'text': actor.slug, 'value': u'team:{}'.format(actor.id)}
```
Comment (MaxBittker) [inline src/sentry/integrations/slack/utils.py]: maybe raise a Not implemented after these cases?

**Review by MaxBittker — APPROVED** [review]
looks ok to me - would you mind writing a test?

File: src/sentry/integrations/slack/utils.py [hunk 2]
```diff
@@ -129,14 +139,8 @@ def build_attachment(group, event=None, tags=None, identity=None, actions=None,
...
-        assignee = GroupAssignee.objects.get(group=group).user
-        assignee = {
-            'text': assignee.get_display_name(),
-            'value': assignee.username,
-        }
-
-        # Add unassign option to the top of the list
-        assignees.insert(0, UNASSIGN_OPTION)
+        assignee = GroupAssignee.objects.get(group=group).assigned_actor()
+        assignee = format_actor_option(assignee)
```
Comment (MaxBittker) [inline src/sentry/integrations/slack/utils.py]: ... maybe do this on one line? maybe not

File: src/sentry/integrations/slack/utils.py [hunk 3]
```diff
@@ -167,15 +171,29 @@ def build_attachment(group, event=None, tags=None, identity=None, actions=None,
...
+    option_groups = []
+
+    if members:
+        option_groups.insert(0, {
+            'text': 'People',
+            'options': members,
+        })
+
+    if teams:
+        option_groups.insert(0, {
```
Comment (MaxBittker) [inline src/sentry/integrations/slack/utils.py]: why not append?

File: src/sentry/integrations/slack/utils.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/slack/utils.py]: IMO it's fine

File: src/sentry/integrations/slack/utils.py (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/slack/utils.py]: 09d2903

**Comment by EvanPurkhiser** [comment]
Updated the action endpoint to do team assignment.

**Pull request merged**